gallery/src/PageHeader.tsx

```diff
@@ -16,12 +16,10 @@ function PageHeader() {
       <div className="nx-page-header__inner">
         <a href="#" className="nx-product">
           <img src={sonatypeLogo} className="nx-product__logo"/>
-          <div className="nx-product__name">
-            Sonatype Component Library
-          </div>
-          <div className="nx-product__version">
-            Version: {packageJson.version}
-          </div>
+          <div className="nx-product__sonatype-wordmark">sonatype</div>
+          <div className="nx-product__vertical-divider">|</div>
```

Comment:
Is there a reason this wasn't done as a border?

Reply:
> Is there a reason this wasn't done as a border?

This worked first try and seemed simpler than applying a border to the box, shrinking the box to match the required height, and then potentially shifting the box around to match the required positioning

Reply:
> This worked first try and seemed simpler than applying a border to the box, shrinking the box to match the required height, and then potentially shifting the box around to match the required positioning

I guess that's alright styling-wise but semantically I'm not a huge fan.  Could you move the pipe character into a `::before` so it isn't in the DOM?

Reply:
> > This worked first try and seemed simpler than applying a border to the box, shrinking the box to match the required height, and then potentially shifting the box around to match the required positioning
> 
> I guess that's alright styling-wise but semantically I'm not a huge fan. Could you move the pipe character into a `::before` so it isn't in the DOM?

Sure: https://github.com/sonatype/sonatype-react-shared-components/pull/30/commits/a9177cf8d66cab40995ef54af80cb59724a78e31